### ISSUE TITLE
Saving autopopulate the filename with the file name in the gridmaker window

### ISSUE THREAD

**Issue by riversn** [bradmartin333/GridMaker#20]
When saving a gridmaker file, can the file name be auto-populated with the filename we have in the box? and if the user changes the save name, that box name should get updated to the new name that was saved?

![image](https://github.com/bradmartin333/GridMaker/assets/101815602/2fa82e1a-0330-491d-96a1-353a2fa1fcec)

**Comment by bradmartin333** [comment]
I agree that is a good feature - but that would require a home directory of sorts

So yes, you could set a home directory and it would remember and then it would load whatever is in the box

This could be problematic if a naming convention is not obeyed - for instance, a special file extension for gridmaker files (not just .xml) or a prefix like GRID_gridname.xml

And then, if you put a name that does not exist, it should keep the last good values - but, if you have GRID_mygrid_2_3 and you want to use that to make GRID_mygrid_2_4 that would require GRID_mygrid_2_ to not exist

now... each file could have a unique hash, but then you would need to have some sort of file history system...

yes, this would be cool for operators, but since this app was intended to be replaced by XferPrint's .xrec system... I can't say I really want to put the work into making this work really well

but, for future programs, I will definitely keep it in mind
